lib/membrane_rtc_engine/endpoints/webrtc/forwarder/encoding_report.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Membrane.RTC.Engine.Endpoint.WebRTC.Forwarder.EncodingReport do
```

Comment:
What about calling this `Forwarder.Status` and renaming `Forwarder.encodings_report` to `Forwarder.get_status()`? I think it would fit better. We could also move `Forwarder.Status` module into `Forwarder` itself.